python/pyproject.toml

```diff
@@ -18,6 +18,6 @@
 [build-system]
 requires = [
     "setuptools>=42",
-    "wheel"
+    "wheel",
```

Comment:
Looks like this file doesn't need to change.

Reply:
Fixed